src/_includes/components/chevron-left.njk

```diff
@@ -1,7 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 26.4.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
If you're going to test for accessibility these generic id's such as Layer_1 are going to cause errors since they're not unique. 
But the question is probably rather if you really need an ID here. :)